packages/charting/src/components/ProgressBarChart/ProgressBarChart.base.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { IProcessedStyleSet } from '../../Styling';
+import { classNamesFunction } from '../../Utilities';
+import { IProgressBarChartProps, IProgressBarChartStyleProps, IProgressBarChartStyles } from './ProgressBarChart.types';
+
+const getClassNames = classNamesFunction<IProgressBarChartStyleProps, IProgressBarChartStyles>();
+
+export class ProgressBarChartBase extends React.Component<IProgressBarChartProps, {}> {
+  public static defaultProps: Partial<IProgressBarChartProps> = {
+    width: 400,
+    height: 50,
+    barHeight: 20
+  };
+  private _classNames: IProcessedStyleSet<IProgressBarChartStyles>;
+  constructor(props: IProgressBarChartProps) {
+    super(props);
+  }
+  public render(): JSX.Element {
+    const { data, width, height, barHeight, chartTitle, theme, className, styles, colors } = this.props;
+    const persentage = (data!.value / data!.total) * 100;
+    this._classNames = getClassNames(styles!, {
+      theme: theme!,
+      width: width!,
+      height: height!,
+      className,
+      barHeight,
+      colors,
+      persentage
+    });
+
+    return (
+      <div className={this._classNames.root}>
+        <div>
+          <span className={this._classNames.value}>
+            <strong>{chartTitle}</strong>
```

Comment:
@dzearing What's the preferred way of referencing Fabric type ramp & styles inside common controls? Want to make sure things like size & weight map to variables so that we're not incurring debt by hard-coding those values.

Reply:
...or maybe @kkjeer can answer the Fabric variables question?

Reply:
@joetheday We are adding a `Text` component which can abstract text rendering based on the default theme.

In the meantime, you can access `theme.fonts.*` and mix them into your styling:

For example see PersonaCoin.styles.ts:

```tsx
return {
    coin: [
      classNames.coin,
      theme.fonts.medium,
      { ... styles }
}
```